app/registries/foreman/settings/provisioning.rb

```diff
@@ -0,0 +1,200 @@
+Foreman::SettingManager.define(:foreman) do
+  category(:provisioning, N_('Provisioning')) do
+    # facts which change way too often
+    IGNORED_FACTS = [
+      'load_averages::*',
+      'memory::system::capacity',
+      'memory::system::used*',
+      'memory::system::available*',
+      'memory::swap::capacity',
+      'memory::swap::used*',
+      'memory::swap::available*',
+    ].freeze
+
+    IGNORED_INTERFACES = [
+      'lo',
+      'en*v*',
+      'usb*',
+      'vnet*',
+      'macvtap*',
+      ';vdsmdummy;',
+      'veth*',
+      'docker*',
+      'tap*',
+      'qbr*',
+      'qvb*',
+      'qvo*',
+      'qr-*',
+      'qg-*',
+      'vlinuxbr*',
+      'vovsbr*',
+      'br-int',
+    ].freeze
+
+    unattended_url = "http://#{SETTINGS[:fqdn]}"
+    owner_select = [{:name => _("Users"), :class => 'user', :scope => 'visible', :value_method => 'id_and_type', :text_method => 'login'},
+                    {:name => _("Usergroups"), :class => 'usergroup', :scope => 'visible', :value_method => 'id_and_type', :text_method => 'name'}]
+
+    setting('host_owner',
+      type: :string,
+      description: N_("Default owner on provisioned hosts, if empty Foreman will use current user"),
+      default: nil,
+      full_name: N_('Host owner'),
+      collection: proc { owner_select },
+      include_blank: N_("Select an owner"))
+    setting('root_pass',
+      type: :string,
+      description: N_("Default encrypted root password on provisioned hosts"),
+      default: nil,
+      full_name: N_('Root password'),
+      encrypted: true)
+    setting('unattended_url',
+      type: :string,
+      description: N_("URL hosts will retrieve templates from during build, when it starts with https unattended/userdata controllers cannot be accessed via HTTP"),
+      default: unattended_url,
+      full_name: N_('Unattended URL'))
+    setting('safemode_render',
+      type: :boolean,
+      description: N_("Enable safe mode config templates rendering (recommended)"),
+      default: true,
+      full_name: N_('Safemode rendering'))
+    setting('access_unattended_without_build',
+      type: :boolean,
+      description: N_("Allow access to unattended URLs without build mode being used"),
+      default: false,
+      full_name: N_('Access unattended without build'))
+    setting('manage_puppetca',
+      type: :boolean,
+      description: N_("Foreman will automate certificate signing upon provision of new host"),
+      default: true,
+      full_name: N_('Manage PuppetCA'))
+    setting('ignore_puppet_facts_for_provisioning',
+      type: :boolean,
+      description: N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"),
```

Comment:
Looks like this is in `app/services/fact_parser.rb` and `app/services/foreman_chef/fact_parser.rb` so not limited to Puppet facts.
```suggestion
      description: N_("Stop updating IP and MAC address values from facts (affects all interfaces)"),
```